daffodil-cli/src/it/scala/org/apache/daffodil/debugger/TestCLIDebugger.scala

```diff
@@ -28,11 +28,9 @@ import org.apache.daffodil.CLI.Util
 
 class TestCLIdebugger {
 
-  val DAFFODIL_JAVA_OPTS = Map("DAFFODIL_JAVA_OPTS" -> "-Xms256m -Xmx2048m -Djline.terminal=jline.UnsupportedTerminal -Dfile.encoding=UTF-8")
-  //  Dubugging tests were not executing under Windows and especially under Eclipse
-  //  due to the use of a non-interactive console.
+  val DAFFODIL_JAVA_OPTS = Map("DAFFODIL_JAVA_OPTS" -> "-Xms256m -Xmx2048m -Dfile.encoding=UTF-8")
   //  Set the DAFFODIL_JAVA_OPTS environment variable for Debugger tests to specify
-  //  the use of an unsupported terminal: -Djline.terminal=jline.UnsupportedTerminal
+  //  the use of a dumb terminal: -Dorg.jline.terminal.dumb
```

Comment:
Should we be specigying terminal.dum in JAVA_OPTS above?

Reply:
No, because I had a hunch that JLine 3 wouldn't need -Dorg.jline.terminal.dumb in DAFFODIL_JAVA_OPTS anyway.  I noticed from my scanning of the JLine 3 sources that JLine 3 goes to some effort to pick the correct terminal implementation (including the dumb terminal) using the TERM environment variable and so on.  I updated the comment to show how to specify the use of a dumb terminal in case we ever need to do it in the future, but we didn't need it to make the Debugger tests pass (although it's also true that the Debugger tests hit only a third of the JLine 3 code paths according to the coverage report).

Reply:
I imagine the completer would be a pain to test with our current infrastrucutre. I'd rather just not worry about it. And I'll remove this entire comment, it doesn't really say anything helpful. I think the DAFFODIL_JAVA_OPTS is line is pretty clear what it's doing..